gulpfile.babel.js

```diff
@@ -99,7 +99,8 @@ gulp.task('styles', () => {
     .pipe($.if('*.css', $.minifyCss()))
     .pipe($.size({title: 'styles'}))
     .pipe($.sourcemaps.write('./'))
-    .pipe(gulp.dest('dist/styles'));
+    .pipe(gulp.dest('dist/styles'))
+    .pipe(browserSync.stream({match: '**/*.css'}));
```

Comment:
Change this line for `.pipe($.if(bs.active, bs.stream({ match: '**/*.css' })));`. Because the gulp task `style` can be executed without Browsersync.